aws-opentelemetry-distro/tests/amazon/opentelemetry/distro/test_aws_metric_attribute_generator.py

```diff
@@ -950,6 +951,19 @@ def test_sdk_client_span_with_remote_resource_attributes(self):
         self._validate_remote_resource_attributes("AWS::Kinesis::Stream", "aws_stream_name")
         self._mock_attribute([AWS_STREAM_NAME], [None])
 
+        # Validate behaviour of AWS_CONSUMER_ARN attribute, then remove it.
```

Comment:
Add a test covering the case when both stream name and consumer ARN are present.

Reply:
Added.